packages/apptive_grid_core/lib/src/network/apptive_grid_client.dart

```diff
@@ -763,3 +711,35 @@ class ApptiveGridClient extends ChangeNotifier {
     notifyListeners();
   }
 }
+
+/// Meta data for a paging request
+class PagingRequest {
```

Comment:
If decided to keep it should be moved out of the client itself and be exposed by the package but I think it is cleaner to separate it

Reply:
removed